requirements.txt

```diff
@@ -1,5 +1,6 @@
-coloredlogs
```

Comment:
It is convention to put these in alphabetical order?

Reply:
If so, my vote would be to add that to pre-commit hook for standardization

Reply:
I would be pretty hesitant to implement this. For well-crafted libraries, import order should not matter. Tensorflow in my experience breaks that assumption, and armory may also.  At this point, I fear that we'd be buying unnecessary headache; when we have more tests this would be less of a concern. 

We also don't have tons of imports (max=22 in __main__.py, median~=3) and PEP-8 doesn't calll for ordering although Google Style and its many descendants do. IMHO this is a java convention for self-defense; I've never understood what it benefits.